### ISSUE TITLE
Timeline Slicer Force Selection

### ISSUE THREAD

**Issue by Talark** [Microsoft/powerbi-visuals-timeline#39]
Hello,
When turning the Current Period option on in Force Selection of the Timeline Slicer, it is visually selecting the current month (we have month granularity) but the report is not reflecting the data related to the current period selection, instead it is showing the data for the old date selection (the date selected in the timeline slicer when the report was published), and also when I click on the filter pane the current period selection is getting back again to it's old selection which is really weird, could you help me with this issue please? is it a bug?

**Comment by uve** [comment]
@Talark yes, it's a bugs and it should be fixed by [this commit](https://github.com/Microsoft/powerbi-visuals-timeline/commit/f10531ac15a6b265e383e0d8f5fc12118378a139)

**Comment by Talark** [comment]
@uve Thanks for the reply, I just want to ask if this commit is pushed to production or not? and when the fix is in production what should I do? Should I recreate the Timeline Slicer in my report in Power BI Desktop and publish it again or I don't need to do anything?

**Comment by adam-wood** [comment]
@uve incidentally, commit #40 changed some behaviour I was using. I used 'Force Selection' to select the current month, but still wanted to be able to allow the user to select other months. Any ideas on how to achieve this now?

**Comment by Talark** [comment]
@uve @adam-wood did you find a workaround or do you have a different suggestion to solve this issue? I am facing the same problem, I cannot get the current month selected, because the force selection (current period) when on, won't let me select any other month in the slicer!!

**Comment by adam-wood** [comment]
@Talark I ended up using a workaround which isn't very scalable. I essentially manually select the correct month for each page in Power BI desktop and re-publish. That way, our users are sent to the current month when they first visit the report. I manually do this on the first of each month 😒.

On an unrelated note, I have also experimented with going back to the standard slicer as this makes it possible to sync the slicer across pages, which is useful for my report. I have a few projects to tie up by September and am hoping to try and dive into the source in October to see if I can contribute.
